sjb/config/common/test_cases/crio.yml

```diff
@@ -128,7 +138,7 @@ post_actions:
       trap 'exit 0' EXIT
       if [[ -n "${JOB_SPEC:-}" ]]; then
         JOB_SPEC="$( jq --compact-output ".buildid |= \"${BUILD_NUMBER}\"" <<<"${JOB_SPEC}" )"
-        docker run -e JOB_SPEC="${JOB_SPEC}" -v "/data:/data:z" registry.svc.ci.openshift.org/ci/gcsupload:latest --dry-run=false --gcs-path=gs://origin-federated-results --gcs-credentials-file=/data/credentials.json /data/gcs/*
+        sudo docker run -e JOB_SPEC="${JOB_SPEC}" -v "/data:/data:z" registry.svc.ci.openshift.org/ci/gcsupload:latest --dry-run=false --gcs-path=gs://origin-federated-results --gcs-credentials-file=/data/credentials.json /data/gcs/*
```

Comment:
Is there no way using `crictl` or something to run this container? Why do we need to install another container runtime? This is crazy

@runcom

Reply:
@stevekuznetsov I agree, and would prefer not to do this.  My understanding (could be totally wrong) is, we'd need to install podman.  Unf. it's packaging isn't entirely stable yet on RHEL, hence my note in the commit message.  We can replace docker with podman at a later date pretty easy.